apps/advisor/pkg/app/utils.go

```diff
@@ -78,7 +78,7 @@ func processCheck(ctx context.Context, client resource.Client, obj resource.Obje
 		return fmt.Errorf("error running steps: %w", err)
 	}
 
-	report := &advisorv0alpha1.CheckV0alpha1StatusReport{
+	report := &advisorv0alpha1.CheckReport{
```

Comment:
just FTR, this change does not work, more info in this [thread](https://raintank-corp.slack.com/archives/C04RVCAG9B5/p1745325966811089)